web/src/components/Button.tsx

```diff
@@ -9,12 +9,14 @@ interface ButtonProps {
   disabled?: boolean;
   size?: SizeOptions;
   shouldShrink?: boolean;
-  type?: 'primary' | 'secondary' | 'tertiary' | 'link';
+  type?: 'primary' | 'secondary' | 'tertiary' | 'link' | 'transparent';
```

Comment:
Maybe this should be called something like glass instead?

Reply:
eeh, I guess that's fine with type checking but it is harder to spell for sure...

I'll change it though.